shapely/tests/legacy/test_equality.py

```diff
@@ -27,21 +26,9 @@ def test_equals_exact():
     assert not p1.equals_exact(p3, 1e-5)
 
 
-def test_almost_equals_default():
-    p1 = Point(1.0, 1.0)
-    p2 = Point(1.0 + 1e-7, 1.0 + 1e-7)  # almost equal to 6 places
-    p3 = Point(1.0 + 1e-6, 1.0 + 1e-6)  # not almost equal
-    with pytest.warns(ShapelyDeprecationWarning):
-        assert p1.almost_equals(p2)
-    with pytest.warns(ShapelyDeprecationWarning):
-        assert not p1.almost_equals(p3)
-
-
-def test_almost_equals():
+def test_almost_equals_removed():
```

Comment:
I suggest removing this test.

Reply:
Sure, it will be noted in the release notes anyways.